config.yml

```diff
@@ -2,36 +2,52 @@
 common:
     host_user: root
     host_password: redhat
-    host_ip: "10.66.8.109"
-    rhvm_fqdn: rhvm41-test.nay.redhat.com
+    host_ip: 10.70.34.33
+    #host_ip: 10.70.37.98
+    rhvm_fqdn: cambridge.lab.eng.blr.redhat.com
+    #rhvm_fqdn: dhcp37-98.lab.eng.blr.redhat.com
     browser: firefox
-    test_build: rhvh-4.1-0.20170914.0
+    test_build: rhvh-4.1-0.20171207.0
 
 subscription:
-    rhn_user: QualityAssurance
-    rhn_password: VHVFhPS5TEG8dud9
-    activation_key: rhevh
-    activation_org: "711497"
-    satellite_ip: "10.73.75.177"
-    satellite_hostname: satellite62.lab.eng.pek2.redhat.com
-    satellite_user: admin
-    satellite_password: redhat
-    ca_path: https://10.73.75.177/pub/katello-ca-consumer-satellite62.lab.eng.pek2.redhat.com-1.0-1.noarch.rpm
+    rhn_user: knarra_grafton_pod
+    rhn_password: graftonqa
```

Comment:
Even we did not notice that this is senstive data for we internal, could you please leave it blank.

Reply:
sure, i can do that